site/app/config/config.yml

```diff
@@ -49,13 +49,17 @@ twig:
     form_themes:
         - 'bootstrap_3_layout.html.twig'
         - 'LibrecoresProjectRepoBundle:Form:fields.html.twig'
+    globals:
+        algolia_app_id: "%env(ALGOLIA_APP_ID)%"
+        algolia_search_key: "%ALGOLIA_SEARCH_KEY%"
 
 # Assetic Configuration
 assetic:
     debug:          "%kernel.debug%"
     use_controller: false
     bundles:
         - LibrecoresProjectRepoBundle
+        - LibrecoresSiteBundle
```

Comment:
How is this change related to the PR? If it's not related please open a separate PR to keep this one focused.